src/main/kotlin/no/nav/familie/ef/personhendelse/inntekt/InntektsendringerService.kt

```diff
@@ -17,8 +17,9 @@ class InntektsendringerService(
     fun harEndretInntekt(inntektshistorikkResponse: InntektshistorikkResponse, ident: String, forventetInntekt: Int?): Boolean {
         // hent alle registrerte vedtak som var på personen sist beregning
         val nyesteRegistrerteInntekt = inntektshistorikkResponse.inntektForMåned(YearMonth.now().minusMonths(1).toString())
+        val nestNyesteRegistrerteInntekt = inntektshistorikkResponse.inntektForMåned(YearMonth.now().minusMonths(2).toString())
 
-        return har10ProsentHøyereInntektEnnForventet(nyesteRegistrerteInntekt, ident, forventetInntekt)
+        return har10ProsentHøyereInntektEnnForventet(nestNyesteRegistrerteInntekt, ident, forventetInntekt) && har10ProsentHøyereInntektEnnForventet(nyesteRegistrerteInntekt, ident, forventetInntekt)
     }
 
```

Comment:
10% i forhold til hva? 10% i forhold til forrige måned for begge ? Dvs
Januar
Februar 10% økning
Mars samme lønn som i februar

Her vil mars ha 10% økning mot januar, men den samme som i februar

Reply:
Sant, slik det virker nå så er det bare sjekk mot forventet inntekt fra forrige måned, som i teorien kan være noe annet to måneder bakover. Må vel nesten gjøre om slik at det ikke blir noen falske positiver for de som har endret forventet inntekt mellom de to månedene det sjekkes på.